Modules/System/Cryptography Management/src/DESCryptoServiceProvider.al

```diff
@@ -0,0 +1,68 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Provides helper functions for the Advanced Encryption Standard.
+/// </summary>
+
+codeunit 50102 "DESCryptoServiceProvider"
+{
+    Access = Public;
+
+    /// <summary>
+    /// Encrypts text with DESCryptoServiceProvider
```

Comment:
The way I read this, it's saying that it uses itself to encrypt text.

Reply:
Fair enough, amended to Encrypts text with DotNet Cryptography.DESCryptoServiceProvider, will make a PR this morning.